### ISSUE TITLE
Build Error with Astro SSR adapters

### ISSUE THREAD

**Issue by Tanish2002** [x64Bits/solid-icons#45]
I'm trying deploy my project on cloudflare via the astro provided [ssr adapter](https://docs.astro.build/en/guides/integrations-guide/cloudflare/). 

Whenever I try to build the project I get this error:
```
✘ [ERROR] No matching export in "node_modules/.pnpm/solid-js@1.7.7/node_modules/solid-js/web/dist/server.js" for import "memo"

    node_modules/.pnpm/solid-icons@1.0.8_solid-js@1.7.7/node_modules/solid-icons/lib/index.js:1:47:
      1 │ import { mergeProps, spread, insert, isServer, memo, template } from 'solid-js/web';
        ╵                                                ~~~~

✘ [ERROR] No matching export in "node_modules/.pnpm/solid-js@1.7.7/node_modules/solid-js/web/dist/server.js" for import "template"

    node_modules/.pnpm/solid-icons@1.0.8_solid-js@1.7.7/node_modules/solid-icons/lib/index.js:1:53:
      1 │ import { mergeProps, spread, insert, isServer, memo, template } from 'solid-js/web';
        ╵                                                      ~~~~~~~~

 error   No matching export in "node_modules/.pnpm/solid-js@1.7.7/node_modules/solid-js/web/dist/server.js" for import "memo"
  File:
    node_modules/.pnpm/solid-icons@1.0.8_solid-js@1.7.7/node_modules/solid-icons/lib/index.js:1:47
  Code:
    > 1 | import { mergeProps, spread, insert, isServer, memo, template } from 'solid-js/web';
        |                                               ^
      2 | import { splitProps } from 'solid-js';
      4 | const _tmpl$ = /*#__PURE__*/template(`<svg stroke-width="0"></svg>`, 2),
```
Out of curiosity I tried it with other adapters like vercel and the problem still persists.

It builds fine without any ssr adapters.

Here is a minimal reproduction of problem: https://codesandbox.io/p/sandbox/bold-wilbur-kch7zp
You can check it by doing a `yarn build`

**Comment by Guergeiro** [comment]
Having the same issue.

**Comment by x64Bits** [comment]
Hi thanks for report this, I'm sure that this problem is related to `exports` in package.json generated by the library, I'm imitating the way solidjs did it in its 1.4 version, but I have seen that they added the `worker` key which I imagine should be used to target the correct import using cloudflare.

**Comment by Guergeiro** [comment]
@x64Bits I've tried to offload it in Astro using `client:only="solid-js"` directives and even just re exporting a the Svg as its separate Solid component, to no avail. Do you have a solution on the way for this?

**Comment by x64Bits** [comment]
Hello, sorry for the delay, I have sent a beta version of the package with a possible correction for this problem, if you can take a look at it for feedback, there is also an open PR in case you have comments #54, greetings and thank you very much.
https://www.npmjs.com/package/solid-icons/v/1.1.0-beta.1

**Comment by x64Bits** [comment]
Hi @silesmo, thanks for let me know about it, [Version 1.1.0](https://www.npmjs.com/package/solid-icons/v/1.1.0) address this issue, can you take a look it is working properly now?

**Comment by x64Bits** [comment]
I'll assume this issue was resolved in 1.1.0, regards.
